package.json

```diff
@@ -101,7 +101,7 @@
   },
   "scripts": {
     "update-schema": "node ./scripts/update-schema",
-    "relay": "relay-compiler --src ./src --schema ./src/schema.graphql --include \"**/*.js\"",
+    "relay": "relay-compiler --src ./ --schema ./src/schema.graphql --include \"**/*.js\"",
```

Comment:
It must be `relay-compiler --src ./src --schema ./schema.graphql --include \"**/*.js\"

And when you run the app or `yarn update-schema`, it should save `schema.graphql` to the root of the project.